### ISSUE TITLE
NXC Wiki on MSSQL Protocol Gives Misinformation and Incorrect Usage Example

### ISSUE THREAD

**Issue by Armageddon0x00** [Pennyw0rth/NetExec#513]
**Describe the bug**
The wiki pages misinforms users on MSSQL protocol. It says the user can do a RID Brute Force with MSSQL protocol. MSSQL can not be directly leveraged for RID brute, the wiki gives misinformation and a not available command on different protocol.

**To Reproduce**
1. Go to nxc wiki page for MSSQL usage and available commands, https://www.netexec.wiki/mssql-protocol/enumerate-users-by-bruteforcing-rid.
2. Try to apply the given command.
3. The `nxc` will throw an error saying this is wrong. Of course it is wrong, this really does not make any sense `nxc mssql 192.168.1.0/24 -u UserName -p 'PASSWORDHERE' --rid-brute`.  This command is exactly the same (only changing the protocol) as SMB one.

Command:` nxc mssql querier.htb.local -u '' -p '' --rid-brute`
Resulted in:
```
usage: nxc [-h] [--version] [-t THREADS] [--timeout TIMEOUT] [--jitter INTERVAL] [--verbose] [--debug] [--no-progress]
           [--log LOG] [-6] [--dns-server DNS_SERVER] [--dns-tcp] [--dns-timeout DNS_TIMEOUT]
           {wmi,rdp,winrm,ldap,ssh,mssql,nfs,smb,ftp,vnc} ...
nxc: error: unrecognized arguments: --rid-brute
```

**Expected behavior**
I expected an error and it occured. Wiki needs to be updated and should be checked for module-protocol confusion on example commands.

**Screenshots**
![image](https://github.com/user-attachments/assets/9efb4b45-1490-402f-a71b-cc37c9e34142)

**NetExec info**
 - OS: Kali
 - Version of nxc: 1.3.0 - NeedForSpeed - 2c5037f
 - Installed from: pipx

**Comment by mpgn** [comment]
Hello,

Can you pull the lastest commit and retry the command ?

Regards

**Comment by Armageddon0x00** [comment]
Yep, this is my bad. Very nice implementation guys. I never thought of running RID brute using MSSQL queries, quite cool!
![image](https://github.com/user-attachments/assets/7b2e7e93-9531-420d-8710-7f2fda4d11ff)
